packages/next/src/middleware/request.ts

```diff
@@ -60,7 +56,16 @@ export class MiddlewareRequest extends Request {
    */
   async next(options?: NextOptions): Promise<MiddlewareResponse> {
     this.applyHeaders()
-    const response = await this.context.next(options)
+    let response = await this.context.next(options)
+
+    // next i18n responses contain a relative location header, and a 301 status
```

Comment:
[dust] this isn't specifically a Next i18n thing: this is our CDN redirecting to a lowercase version of any static file request. In this scenario the i18n file is statically-generated, so the cdn redirects to the lowercase version